toxcore/tox.api.h

```diff
@@ -286,7 +286,10 @@ const FILE_ID_LENGTH              = 32;
  */
 const MAX_FILENAME_LENGTH         = 255;
 
-
+/**
+ * Max legnth of names queried by toxcore
+ */
+const MAX_QUERY_NAME_LENGTH       = 255;
```

Comment:
it's limited to 255 because that's a single u8, so that when we add additional information/meta data, we can include the size of the name vs other data.